CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changes
 
+## edge-21.3.4
+
+This edge release is a release candidate for stable 2.10.1 and fixes
+some issues around publishing of CLI binary for Apple Silicon
+M1 Chips. This release also includes some improvements to the
+destination, and the CLI.
+
+* Updated the IP Watcher in destination to ignore pods in "Terminating" state
+  (thanks @Wenliang-CHEN!)
+* Added `installNamespace` toggle in the jaeger extension's install.
+  (thanks @jijeesh!)
+* Updated `healthcheck` pkg to have `hintBaseURL` configurable, useful
```

Comment:
```suggestion
* Updated `healthcheck` pkg to have `hintBaseURL` configurable. This allows
  external extensions to use the pkg for development.

Reply:
Hmm 🤔 We were allowing that previously too. This just allows `hintBaseURL` configurable.